src/Containers.jsx

```diff
@@ -864,7 +864,7 @@ class Containers extends React.Component {
                                                     <CardTitle>
                                                         <Flex justifyContent={{ default: 'justifyContentFlexStart' }}>
                                                             <h3 className='pod-name'>{caption}</h3>
-                                                            <span>{_("pod group")}</span>
+                                                            <span>{isPodService ? _("pod service group") : _("pod group")}</span>
```

Comment:
"pod service"? ("pod group" is redundant -- a pod is already a container for multiple things (both peas and dolphins -- but that's existing code already, unrelated to this PR)

Reply:
Let's rename after https://github.com/cockpit-project/cockpit-podman/pull/2050 lands